### ISSUE TITLE
wish this project can continue.

### ISSUE THREAD

**Issue by jaypatelani** [etoile/EtoileFoundation#7]
Any future revival plans?
Thanks.

**Comment by davidchisnall** [comment]
I'm not sure about the rest of the team but my efforts are currently focused on [Verona](https://github.com/Microsoft/Verona), which I think is a better base than Objective-C for the kinds of things that I wanted to do with Etoile.

**Comment by jaypatelani** [comment]
nim or ada can be also good base for Etoile.

**Comment by davidchisnall** [comment]
Not really. Neither has a language level abstraction for compartmentalisation or for data distribution on NUMA systems.

**Comment by jaypatelani** [comment]
Best of luck for future development. Looking forward to updates :)
